### PR TITLE
[docs] Add the February 2021 changelog.

### PR DESCRIPTION
**Pull request by mikenikles**
Related to #3328.

A first, lightweight changelog for the February release with a focus on two user-facing features:
* Synchronize Theia user settings with VS Code
* Deploy Gitpod in an air-gapped network

Please let me know if I missed anything 🙏.

_Note_: For the next release, #2682 will automate some aspects of the changelog. At that time, a full list of all PRs will be included at the end of the release notes.

### CODE REVIEW

**Comment by csweichel** [comment]
> Deploy Gitpod in an air-gapped network
Not sure I'd highlight that. This is not so much an official feature as it is a "low hanging fruit".

Other things in this release:
- improved workspace startup time in high-load situations (ghost workspaces)
- started to [adopt the controller framework](https://kubernetes.io/docs/concepts/architecture/controller/) which will lead to Gitpod producing less load on the Kubernetes API
- fixed env var parsing in context URLs: https://github.com/gitpod-io/gitpod/commit/3a0ca242f2281efe284c434ee4a3b188bd422d0a
- removed the `privileged` flag which is now completely replaced with user-namespaced workspaces: https://github.com/gitpod-io/gitpod/commit/5b6325f0de76c9d78aca768e39f341e5621d0068
- improved workspace startup behaviour/feedback for broken workspace images: https://github.com/gitpod-io/gitpod/commit/e33ff3369dda3934ee16c805170de2d72963a108
- reduced number of daemonSets Gitpod needs: https://github.com/gitpod-io/gitpod/commit/c47c7e1933f9e2340def35e23593fba60d608fd2 https://github.com/gitpod-io/gitpod/commit/61b1b9b38e8c60bc07211cc550c3ebc40434c588
- made registry facade standard - which means all GP installations require HTTPS now: https://github.com/gitpod-io/gitpod/commit/1046faf0f9121eed6a765a62a6d008a5aff804c5

File: docs/changelog/2021-02/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Miscellaneous
+
+* [#3087](https://github.com/gitpod-com/gitpod/pull/3087) - Remove the privileged feature flag. Thanks to [@csweichel](https://github.com/csweichel), [@akosyakov](https://github.com/akosyakov)
+* [#3175](https://github.com/gitpod-com/gitpod/pull/3175) - Fix Env Var context parsing. Thanks to [@AlexTugarev](https://github.com/AlexTugarev), [@csweichel](https://github.com/csweichel)
+* [#3177](https://github.com/gitpod-com/gitpod/pull/3177) - [supervisor] Let supervisor fail when first IDE start fails. Thanks to [@corneliusludmann](https://github.com/corneliusludmann), [@csweichel](https://github.com/csweichel)
+* [#3182](https://github.com/gitpod-com/gitpod/pull/3182) - [registry-facade] Remove feature flag. Thanks to [@csweichel](https://github.com/csweichel), [@corneliusludmann](https://github.com/corneliusludmann)
+* [#3228](https://github.com/gitpod-io/gitpod/pull/3228) - Allow air-gap Gitpod installations. Thanks to [@corneliusludmann](https://github.com/corneliusludmann), [@geropl](https://github.com/geropl)
+* Improved workspace startup time in high-load situations
```
Comment (csweichel) [inline docs/changelog/2021-02/README.md:22]: Thanks to @geropl

File: docs/changelog/2021-02/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+* Started to [adopt the controller framework](https://kubernetes.io/docs/concepts/architecture/controller/) which will lead to Gitpod producing less load on the Kubernetes API
```
Comment (csweichel) [inline docs/changelog/2021-02/README.md:23]: Thanks to @aledbf

**Review by csweichel — APPROVED** [review]
Other than the two missing attributions, LGTM

**Comment by mikenikles** [comment]
Thanks Chris for mentioning the authors of the two last items 👍.

**Pull request merged**